components/GitHubEvent.tsx

```diff
@@ -7,38 +8,47 @@ import GitHubEventAction from './GitHubEventAction';
 // and https://docs.github.com/en/rest/reference/activity#list-public-events
 // and https://docs.github.com/en/developers/webhooks-and-events/github-event-types
 interface GitHubEvent {
-  id: string
-  type: string
-  actor: GitHubActor
-  repo: GitHubRepo
-  created_at: string
-  payload: any
+  id: string;
+  type: string;
+  actor: GitHubActor;
+  repo: GitHubRepo;
+  created_at: string;
+  payload: any;
 }
 
 interface GitHubActor {
-  id: number
-  login: string
-  display_login?: string
-  gravatar_id: string
-  url: string
-  avatar_url: string
+  id: number;
+  login: string;
+  display_login?: string;
+  gravatar_id: string;
+  url: string;
+  avatar_url: string;
 }
 
 interface GitHubRepo {
-  id: number
-  name: string
-  url: string
+  id: number;
+  name: string;
+  url: string;
 }
 
 function GitHubEvent(props: GitHubEvent): JSX.Element {
-  const {type, actor, repo, payload} = props;
-  const userLink = !actor.login.includes('[bot]') ? <ELink link={`https://github.com/${actor.login}`}>{`@${actor.login}`}</ELink> : actor.login;
+  const { type, actor, repo, payload, created_at } = props;
+  // stores formatted timestamp for event creation timestamp using moment's fromNow() method
+  const eventTimestamp = moment(created_at).fromNow();
+  const userLink = !actor.login.includes('[bot]') ? (
+    <ELink
+      link={`https://github.com/${actor.login}`}
+    >{`@${actor.login}`}</ELink>
+  ) : (
+    actor.login
+  );
   return (
     <>
       {/* <div className="card" style={{marginTop: "20px"}}> */}
       {/* <div className="card-body"> */}
       {userLink} <GitHubEventAction type={type} payload={payload} />{' '}
       <ELink link={`https://github.com/${repo.name}`}>{repo.name}</ELink>
+      <span style={{ float: 'right' }}>{eventTimestamp}</span>
```

Comment:
You may want to define a class to control this `<span>`'s behaviour, especially since I think you'll need some more CSS to handle responsiveness!